src/styles/index.scss

```diff
@@ -1,3 +1,219 @@
 body {
-  margin: 0;
+  margin: 0px;
+  padding: 0px;
+
+  font-family:  Roboto, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
+
+a{
+   text-decoration: none;
+   color: inherit;
+}
+
+ol, ul{
+  list-style: none;
+}
+
+.header{
+  display: flex;
+  justify-content: space-around;
+
+  box-shadow: 0px 2px 4px 0px #0000000D;
+}
+
+.logo{
+  margin-top: 10px;
+}
+
+.nav__list{
+  margin: 0px;
+  padding: 0px;
+  display: flex;
+  gap:20px
+}
+
+.nav__item{
+  display: flex;
+  align-items: center;
+  
+  height: 60px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-transform: uppercase;
+
+  transition: 300ms;
+}
+
+.nav__item:hover{
+  color: #00ACDC;
+}
+
+.active{
+position: relative;
+
+color:#00ACDC;
+}
+
+.active::after {
+content: "";
+position: absolute;
+bottom:0;
+
+height: 4px;
+width: 100%;
+
+border-radius: 2px;
+background-color:#00ACDC;
+}
```

Comment:
this color is used 6 times, it would make more sense to move this color to a CSS-variable